Mdzaidsiddique/Cypress
# Initial setup for Cypress: Test cases for title verification and element selection

### Overview
This PR introduces basic Cypress test cases and covers CSS locators. The following changes are included:

### Changes
- Added basic Cypress test cases to verify page title and elements using `cy.get`.
- Implemented tests to cover CSS locators for various elements.
- Ensured that all tests run successfully with initial setup.

### How to Test
1. Clone the repository and checkout the `cypress-automation` branch.
2. Run `npm install` to install dependencies.
3. Run `npx cypress open` to open the Cypress test runner.
4. Execute the test cases to verify functionality.

### Additional Information
- Cypress configuration files and initial test cases are included.
- Future work: Expand test coverage for more complex scenarios and add more CSS locator tests.
